### PR TITLE
K8SPXC-288 eks quickstart - initial doc

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPXC-288](https://jira.percona.com/browse/K8SPXC-288)

### CODE REVIEW

File: source/eks.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+   it `following the official installation instructions <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+
+Create a new IAM role with EKS permissions
+===========================================
+
+The first thing to use is configuring the `AWS-IAM-Authenticator <https://github.com/kubernetes-sigs/aws-iam-authenticator>`_ to allow IAM authentication with your Kubernetes cluster.
+
+Open the IAM console, select Roles on the left and then click the Create Role button at the top of the page.
+From the list of AWS services, select EKS and then Next: Permissions at the bottom of the page.
+Leave the selected policies as-is, and proceed to the Review page. Enter a name for the role (e.g. eksrole) and hit the Create role button at the bottom of the page to create the IAM role. The Role ARN specified on the page is to be used later.
```
Comment (delgod) [inline source/eks.rst:28]: is it really needed?
I think it is just needed to configure aws credentials according to https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html doc is enough

File: source/eks.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+   $ eksctl create cluster -f ~/cluster.yaml
+
+
+**is it needed?**
+
+Now update your ``kubectl`` configuration to allow it communicating with the new cluster. You can do it using the AWS CLI ``update-kubeconfig`` command. Run it with the correct cluster-name and region:
+
+.. code:: bash
+
+   $ aws eks --region eu-west-2 update-kubeconfig --name test-cluster
```
Comment (delgod) [inline source/eks.rst:90]: not sure if it is needed

File: source/eks.rst [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+
+Install the Operator
+=======================
+
+1. IAM? Is it already used by kubectl? What about creating Roles and RoleBindings?
```
Comment (delgod) [inline source/eks.rst:96]: I think everything should be fine after `eksctl create cluster -f ~/cluster.yaml` command

File: source/eks.rst [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+   .. code:: yaml
+
+     ...
+     data:
+       ...
+       root: cm9vdF9wYXNzd29yZA==
+
+   Here the actual password is base64-encoded, and ``echo 'cm9vdF9wYXNzd29yZA==' | base64 --decode`` will bring it back to a human-readable form.
+
```
Comment (delgod) [inline source/eks.rst:165]: should we add step like 

```
kubectl port-forward svc/example-proxysql 3306:3306 &
mysql -h 127.0.0.1 -P 3306 -uroot -pNt9YZguajW7nfVXTTrP
```
to show how to connect to mysql?

File: source/eks.rst [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+   nodeGroups:
+       - name: ng-1
+         minSize: 3
+         maxSize: 5
+         instancesDistribution:
+           maxPrice: 0.15
+           instanceTypes: ["m5.xlarge", "m5.2xlarge"] # At least two instance types should be specified
+           onDemandBaseCapacity: 0
+           onDemandPercentageAboveBaseCapacity: 50
+#           spotInstancePools: 2
```
Comment (hors) [inline source/eks.rst:56]: Is it a typo?

**Pull request merged**